jest.config.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
     '<rootDir>/jest.setup.afterenv.js'
   ],
   'transformIgnorePatterns': [
-    'node_modules/(?!react-native|@react-native-community/async-storage|rn-fetch-blob|@integreat-app/integreat-api-client|antd|rc-.+)'
+    'node_modules/(?!react-native|@react-navigation/native/dist|react-navigation-stack|@react-native-community/async-storage|react-navigation-header-buttons|rn-fetch-blob|@integreat-app/integreat-api-client|antd|rc-.+)'
```

Comment:
Why are these ignores needed? ES versions shouldn't be the reason, right? Never touched this path afaik.
If applicable add some comments here.

Reply:
Yes, ES version is the reason. In @react-navigation there are some imports that require files under @react-navigation/native/dist, which are not transcribed correctly. Same for react-navigation-stack and react-navigation-header-buttons.